### ISSUE TITLE
[Bug] Permanent LB/TAB in top left corner 

### ISSUE THREAD

**Issue by CaptainHugg** [RoseTheFlower/UltrawideIndex#1]
Hi,

thank you very much for this patcher! Is there any way to get rid of the permanent display of the x'ed LB button / TAB key in the top left corner? This only occurs when using this patch and widescreen resolutions. The issue is that it covers up the location popup :(

Game version: Xbox Store - Game Pass

![screenshot 455](https://user-images.githubusercontent.com/99926029/159127698-e68f5fde-6b46-4e43-bf53-a6dcd271b0f4.png)

**Comment by RoseTheFlower** [comment]
Hi,

The UI element is coded in a way that it's not entirely invisible when unused, but rather moved out of the 16:9 space and then brought back into the view as scripted. It's a questionable practice but it's not unheard of and you would see similar things in [Young Souls](https://github.com/RoseTheFlower/UltrawideIndex/releases/tag/youngsouls) where the autosave icon is always visible. I also saw it when looking into Disco Elysium or when playing through Jurassic Park with the [TellTaleWidescreenPatcher](https://github.com/PhantomGamers/TellTaleWidescreenPatcher).

I could look into removing it, but locating a small UI element in the memory could be like searching for a needle in a haystack. Removing it would also mean that it would probably be gone from the inventory as well, and I'm not sure that it's the optimal solution for new players.

**Comment by CaptainHugg** [comment]
Fair enough, thank you for this detailed response. That makes a lot of sense.

Well, my current workaround is to pause the game whenever entering a new area as the location banner is perfectly visible then. Thanks again! :)

**Comment by RoseTheFlower** [comment]
Could you share your save file from %LOCALAPPDATA%\Packages\Finji.TUNIC_tys0ffscxatjj\SystemAppData\wgs ? I need to see the location text in order to try and adjust for it. It would also help if you could tell me where to go for it to appear.

**Comment by CaptainHugg** [comment]
Absolutely! Here you go, hope that helps:

[tunic-savegame.zip](https://github.com/RoseTheFlower/UltrawideIndex/files/8309734/tunic-savegame.zip)

**Comment by dewdrinker19** [comment]
Hi guys, is there any solution to this?  Amazing work on this patch.  That LB icon appears all the time except in a menu.  Thank you.

**Comment by jordandef** [comment]
I am having the same issue.

**Comment by TriForceX** [comment]
Same issue in GOG version 2020.3.17.21367

Using 1.0 patch it works but with the permanent LB icon problem
![image](https://user-images.githubusercontent.com/16083854/167342555-77281164-0e0b-46c1-a0c0-af5471f284ae.png)

Latest 1.1 patch didnt work, still black bars but i noticed the main image got stretched
![image](https://user-images.githubusercontent.com/16083854/167342444-334fb3f8-ff7c-4083-90dd-8a57d9ec0022.png)

**Comment by keatonthewise** [comment]
I love playing this game ultrawide, but this is the only issue I have. Kinda distracting for me personally, but I understand if it's a hard thing to narrow down, @RoseTheFlower.

**Comment by HarryMuscle** [comment]
I uploaded a fix for this issue to Nexus Mods for anyone interested:

https://www.nexusmods.com/tunic/mods/51

It required editing the data.unity3d file where the game stores all of its assets and is a pretty large file (hence why I uploaded it to Nexus Mods).
